legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/pureToSQLQuery/tests/testPureToSql.pure

```diff
@@ -288,4 +288,38 @@ function <<test.Test>> meta::relational::tests::functions::pureToSqlQuery::addDr
    let columnNames = $tdsSelectSqlQuery.columns->cast(@Alias).name;
    assert($columnNames->contains('"ID"'));
    assert($columnNames->fold({c, r | $r && $pathNames->contains($c)}, true));
-}
\ No newline at end of file
+}
+
+function <<test.Test>> meta::relational::tests::functions::pureToSqlQuery::testImportDataFlow() : Boolean[1]
+{
+   let runtime = meta::relational::tests::testRuntime();
+   let mapping = meta::relational::tests::simpleRelationalMapping;
+   let debugContext = noDebug();
+   let extensions = meta::pure::router::extension::defaultRelationalExtensions();
+   
+   let tables = meta::relational::tests::dbInc.schemas.tables;
+   let personTable = $tables->filter(t| $t.name == 'personTable')->toOne();
+   let personAddressIdCol = $personTable.columns->cast(@Column)->filter(c| $c.name == 'ADDRESSID')->toOne();
+   let firmTable = $tables->filter(t| $t.name == 'firmTable')->toOne();
+   let firmAddressIdCol = $firmTable.columns->cast(@Column)->filter(c| $c.name == 'ADDRESSID')->toOne();
```

Comment:
We can avoid filters by using existing qualifiers
$db.schema('schemaName').table('tableName').column('columnName')